docs/reference/custom resources/key-management-providers.md

```diff
@@ -130,7 +131,7 @@ spec:
 
 - If a key/certificate is in disabled state, KMP resource creation will FAIL. Users must remove reference to a disabled Key/Certificate or re-enable in Azure Key Vault.
 
-- Ratify does NOT yet support periodic refresh and polling of certificates/keys. If the default latest version changes, object is disabled/expired, or deleted, Ratify will only become aware once the KMP resource is reconciled (edited, deleted, added).
+- Ratify supports periodic refresh and polling of certificates/keys from Azure Key Vault. The `refreshInterval` field can be set to a time duration to refresh the certificates/keys. When no version of the certificate or key is specified, the latest version will be fetched and the resource will be updated. However, if a version is specified, the resource will be locked to that version and will not be updated.
```

Comment:
we might also want to talk about limitation that auto fetching latest  might result in verification failure if previous image signed are with other version of the cert. We can also link N version issue so reader is aware of the road map.

Reply:
Good call out, I'll add that limitation to the doc. :)

Reply:
@susanshi do you have a link to that issue? Or does one need to be created since the n-verion feature isn't included in the refresher for the 1.3 release?

Reply:
I didn't find the n-version issue, @duffney could you help create one?

Reply:
Certainly, here's a link. https://github.com/ratify-project/ratify/issues/1751 :)